test/utils/getLikelyPage.js

```diff
@@ -5,6 +5,7 @@ const LikelyPage = {
     NO_AUTOINIT: 'no-autoinit.html',
     NO_AUTOINIT_MULTIPLE: 'no-autoinit-multiple.html',
     ISSUE_67: 'issues/67.html',
+    ISSUE_151: 'issues/151.html',
```

Comment:
Those names for the tests and pages aren't great. Let's please use some more descriptive as `UPDATE_OPTIONS: 'update-options.html'`.
I will eventually refactor that 61 too.